### DISCUSSION TITLE
「开源之史」系列之八/九：Apache 软件基金会的诞生与崛起 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#80] [Announcements]
# 「开源之史」系列之八/九：Apache 软件基金会的诞生与崛起 - 「开源之道」

https://opensourceway.community/posts/history-of-open-source/07-009-apache-foundation-born/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)
没错, 当年 WebSphere 内置 Apache 才是大家学习 Apache 的信心动力,
不过, 真的没想到一切都是这么顺理成章...

回头来看, 很可能, 最重要的因素:

- 所有推动这事儿落实的人, 都是原先 Apache 列表的创始人
- 在所有关键组织内部, 又都有原先 Apache 创始小组的人
- 在所有关键时间节点上, 又刚好都有现成的案例以及法律规划
- IBM 真心嘦弯腰拣现成的就好...

不过, 真心偏见, 这种弯腰太难得, 也可能当时几乎没感觉, 因为市场太小...

当年的 LAMP 组合, 无论在开源界多知名, 在政府项目中, 可都是 M$/SUN 方案最流行...


> 品牌胜过分叉

这个断言, 从现在看, 可以翻译/理解为: "流量大于品质",
无论现在是否有人使用 Apache 作为 HTTPD , 都不影响, Apache 基金会的江湖地位了...
当年被美国佬亲手灭绝的 印地安羽毛, 现在变成最活跃的互联网创新策源地...

历史就这么反讽...

Brian Behlendorf > ...我选择阿帕奇这个名字是取其积极含义。阿帕奇族是最后一个屈服于美国政府的民族。当时我们担心大公司迟早会参与竞争并‘教化’这块最早的网络之地，所以在我看来，阿帕奇是个很好的名称，也有人说这个词一语双关-因为正如Apache（与"a patchy"谐音）的名字所表明的那样，他们确实是在给服务器打补丁
